toxav/ring_buffer.h

```diff
@@ -0,0 +1,18 @@
+#ifndef RING_BUFFER_H
```

Comment:
OH... I've been reading it as ring, as in ringing call... too much AV on the brain